src/modules/netinstall/PackageModel.cpp

```diff
@@ -170,6 +170,38 @@ PackageModel::headerData( int section, Qt::Orientation orientation, int role ) c
     return QVariant();
 }
 
+/** @brief Sets the checked flag on matching groups in the tree
+ *
+ * Recursively traverses the tree pointed to by m_rootItem and
+ * checks if a group name matches any of the items in @p selectNames.
+ * If a match is found, set check the box for that group and it's children.
+ *
+ * Individual packages will not be matched.
+ *
+ */
+void
+PackageModel::setSelections( QStringList selectNames )
+{
+    if ( m_rootItem )
+    {
+        setSelections( selectNames, m_rootItem );
+    }
+}
+
+void
+PackageModel::setSelections( QStringList selectNames, PackageTreeItem* item )
```

Comment:
This is an implementation detail, and doesn't need to be a member function at all (it does not refer to any member variables). It could be a static free function instead.